catalog/core/catalog-core-validator/src/main/java/ddf/catalog/validation/impl/validator/ISO3CountryCodeValidator.java

```diff
@@ -38,10 +37,11 @@
   private final boolean ignoreCase;
 
   private static final Set<String> COUNTRY_CODES =
-      Arrays.stream(Locale.getISOCountries())
-          .map(iso2CC -> new Locale("", iso2CC))
-          .map(Locale::getISO3Country)
-          .map(String::toUpperCase)
+      StandardRegistryImpl.getInstance()
+          .lookup("ISO3166", "1")
+          .getStandardEntries()
```

Comment:
✏️ Since lookup can return null, do we need to do any verification its not null?

Reply:
I moved this logic to the constructor - at least this way the program has a chance of recovering if lookup returns null.